packages/xstate-react/src/createActorContext.tsx

```diff
@@ -1,14 +1,12 @@
 import * as React from 'react';
-import { createContext, useContext } from 'react';
 import { useInterpret } from './useInterpret';
-import { useActor } from './useActor';
-import { useSelector } from './useSelector';
+import { useActor as useActorUnbound } from './useActor';
+import { useSelector as useSelectorUnbound } from './useSelector';
 import { ActorRefFrom, AnyStateMachine, EmittedFrom, EventFrom } from 'xstate';
 
 export function createActorContext<TMachine extends AnyStateMachine>(
   machine: TMachine
-): React.Context<ActorRefFrom<TMachine>> & {
```

Comment:
I don't think that we need to intersect with `React.Context` here, we can slightly adjust the return type's shape (we extend its structure anyway).